recipe/meta.yaml

```diff
@@ -12,23 +12,31 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py>=310 or s390x]
```

Comment:
could I ask, why is this?

Reply:
ok, could you add a note saying that this is because `numba` isn't available for `python 3.11` at the time of building? When it is available, a future maintainer will want to remove that selector

Also, is there an inherent technical reason why this feedstock isn't feasible for `s390x`? If not, please remove the `s390x` selector and just delete it from the snowflake channel. Because the recipe should reflect only matters related to technical stuff about the recipe - not customer delivery requirements. The feedstocks can be built by external people, which could include people wanting to build for `s390x`

Reply:
yes, numba is skipped for s390 altogether

Reply:
ok thanks

Reply:
would you be able to add such a note at all please? Otherwise all good